lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb

```diff
@@ -0,0 +1,15 @@
+Sequel.migration do
+  up do
+    alter_table(:dynflow_steps) do
+      if Sequel::Model.db.database_type == :mysql
```

Comment:
How do I set this `@db`? It does not seem to be present in migration block. I suppose I should ask world instance but I'm not sure how to get it in migration.